### PR TITLE
Better Save and Load

### PR DESCRIPTION
**Pull request by drg101**
- Current state of intersections is saved
- State of each layer being "expanded" is saved
- State is converted & compressed to base64, which should make it cleaner & smaller to share

Make sure to save a new state before you load one of the old ones from the previous branch since this one decompresses it.

### CODE REVIEW

**Review by honeybadgerofdoom — APPROVED** [review]
Everything works on my end, solid work.

**Review by Marc-9 — APPROVED** [review]
Thanks for the intersection change looks good to me however unrelated to this PR it weird that some of the advance constraints have a slider however the values are only 0 || 1

Example: SVI advance constraint EPL MOBILE

**Review by pierce-smith1 — APPROVED** [review]
LGTM

File: src/js/ui/NewDataExploration/Workspace.js [hunk 1]
```diff
@@ -97,23 +104,31 @@ export default React.memo(function Workspace() {
...
+        if(deSerializedWorkspace.intersect != null) {
+            setIntersect(deSerializedWorkspace.intersect)
+        }
+        const collections = new Set(deSerializedWorkspace.layers.map(e => e.collection))
         setWorkspace(layers.map(layer => {
             const isIn = collections.has(layer.collection);
             if(isIn) {
-                const deSerializedLayer = deSerializedWorkspace.find(e => e.collection === layer.collection);
-                layer.on = deSerializedLayer.on;
+                const deSerializedLayer = deSerializedWorkspace.layers.find(e => e.collection === layer.collection);                layer.on = deSerializedLayer.on;
```
Comment (pierce-smith1) [inline src/js/ui/NewDataExploration/Workspace.js:130]: You might want to fix this formatting issue though

Comment (drg101) [reply src/js/ui/NewDataExploration/Workspace.js:130]: Not sure how this happened, will fix in next PR.

**Pull request merged**